hermes/probe/probe.go

```diff
@@ -16,7 +16,7 @@
 // - Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
 // - Evan Spendlove, GitHub: evanSpendlove.
 //
-// Monitor implements a probe that monitors a storage system using the
+// probe implements a probe that monitors a storage system using the
```

Comment:
This sounds very similar as what's in line 22. Just keep one: either line 19-20 or line 22.

Reply:
I've removed this in the latest commit.